Sources/ToucanModels/ContentConfigurations/ParagraphStyles.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ParagraphStyles.swift
+//
+//  Created by gerp83 on 2025. 03. 28.
+//
+    
+
+public struct ParagraphStyles: Decodable, Equatable {
+    
+    enum CodingKeys: CodingKey {
+        case note
+        case warn
+    }
+    
+    public var note: [String]
+    public var warn: [String]
+
+    // MARK: - defaults
+
+    public static var defaults: Self {
+        .init(
+            note: ["note"],
+            warn: ["warn", "warning"]
```

Comment:
We should use the data from the `HTMLVisitor` file, also we should pass the values down there via the renderer.

```
"note": ["note"],
"warning": ["warn", "warning"],
"tip": ["tip"],
"important": ["important"],
"error": ["error", "caution"],
```